x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/expandable_section/expandable_section.scss

```
@@ -5,3 +5,9 @@
 .mlExpandableSection-contentPadding {
   padding: $euiSizeS;
 }
+
+// Make sure the charts tooltip in popover
+// have higher zIndex than Eui popover cells
+[id^='echTooltipPortal'] {
+  z-index: 10000 !important;
```

Assuming this achieves the same result, consider changing to an EUI z-index variable.

```suggestion
  z-index: $euiZLevel9 !important;
```

---

Updated here https://github.com/elastic/kibana/pull/91061/commits/3f25370624a2c29cbe605deeb5a840341785b57c